content/donate/privacy_policy.md

```diff
@@ -0,0 +1,12 @@
+# Donation Privacy Policy
+Your privacy will receive the utmost respect. The details of your donation will not be shared, without your express knowledge and consent. This includes your name, email address, donation amount, how many times you've donated, or if you are a monthly subscriber.
+
+Donors who have a yearly total of $50 USD, an equivalent EUR amount, or more may choose to have their name displayed on the Public Sponsors page. The intention is to provide additional incentive to people who appreciate such disclosure as a form of public recognition for their well-appreciated and valuable contributions to the Haiku Project. By default, your name will not be displayed.
```

Comment:
we don't do this anymore, so I would omit this part unless we actually plan to do it again.